src/app/components/Account/index.tsx

```diff
@@ -160,3 +169,85 @@ export const Account: FC<AccountProps> = ({ account, token, isLoading, tokenPric
     </>
   )
 }
+
+type ConsensusAccountDataProps = {
+  account?: Account
+  isLoading: boolean
+  showLayer?: boolean
+  standalone?: boolean
+}
+
+export const ConsensusAccountData: FC<ConsensusAccountDataProps> = ({
+  account,
+  isLoading,
+  showLayer,
+  standalone,
+}) => {
+  const { t } = useTranslation()
+  const { isMobile } = useScreenSize()
+
+  if (isLoading) return <TextSkeleton numberOfRows={7} />
+  if (!account) return null
```

Comment:
We should probably better handle this? There is no account with this address, or something like that?

Reply:
1. I didn't write this code now, only moved this around.
2. I am not even sure if this ever happens.
   - If the address is invalid, we get a different error.
   - If we can't load because of network issues, we get a different error
   - If the data is still showing, we get a different error.

So I think this is probably a dead codepath. However, if we remove it, then TS will complain about `account` being possibly undefined later on...